web/src/app/components/brand/box-logo/box-logo.component.html

```diff
@@ -0,0 +1,16 @@
+<svg
+  width="300"
+  height="240"
```

Comment:
Wait, why are you "hard coded" those values but even use width and height properties in you `component.ts` ?